frontend/integration-tests/tests/kubevirt/node.maintenance.scenario.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-undef */
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+
+// eslint-disable-next-line no-unused-vars
+import { waitForCount, removeLeakedResources, VM_BOOTUP_TIMEOUT, PAGE_LOAD_TIMEOUT, POD_START_TIMEOUT } from './utils';
+import { examplePod } from './mocks';
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded, filterForName, resourceRows } from '../../views/crud.view';
+import { overviewTab } from '../../views/kubevirt/virtualMachine.view';
+import { detailViewAction } from '../../views/kubevirt/vm.actions.view';
+import Pod from './models/pod';
+import * as podView from '../../views/kubevirt/pod.view';
+
+
+describe('Test Node Maintenance Mode', () => {
```

Comment:
shouldn't we also test the node status in `/k8s/cluster/nodes`?

Reply:
Added